docs/plugins/subsonicupdate.rst

```diff
@@ -30,5 +30,6 @@ The available options under the ``subsonic:`` section are:
 
 - **url**: The Subsonic server resource. Default: ``http://localhost:4040``
 - **user**: The Subsonic user. Default: ``admin``
+- **auth**: authentification method token or encoded password
```

Comment:
I think this needs to be expanded: For me, this sounds as if the value should be the token or password, rather than just an indication of the authentication method. The valid choices (`token` and `enc`) should be indicated, and the description of the `pass` key below should reflect which values it can take in each case.

Since the `enc:` prefix seems to be allowed for tokens, too, maybe it would be better to rename this option to `plain`?

Reply:
change enc by plain